tools_iuc.yaml

```diff
@@ -3391,3 +3391,26 @@ tools:
   - name: pe_histogram
     owner: iuc
     tool_panel_section_label: Graph/Display Data
+  
+  - name: metaphlan
```

Comment:
I have 12 new repos for human related tools ...

Reply:
I wanted to add with a new PR :)

Reply:
a 14 ...